README.md

```diff
@@ -1,703 +1,48 @@
-<p align="center">
-  <img src="https://github.com/lucasbento/create-graphql/raw/master/content/logo.png">
-</p>
+# ![Create-GraphQL Logo](/media/logo.png)
 
-<h1 align="center">Create GraphQL</h1>
+# Create-GraphQL
```

Comment:
I think it looks weird with the logo, maybe it would look better without it.

For now we can keep the HTML tags.

Maybe remove the `#` of the logo would be better as well.